### ISSUE TITLE
Tooltips broken

### ISSUE THREAD

**Issue by endeavour** [nwolverson/vscode-ide-purescript#8]
Tooltips are not appearing when I hover over symbols. To complicate things somewhat I'm running VS Code on Ubuntu 16.04 inside a virtualbox guest on a windows 10 host.

**Comment by nwolverson** [comment]
Do you get autocomplete? The most common issue is not getting any kind of type information, which means some kind of issue with psc-ide-server. There should be some kind of popup on start which tells you if that process started.

Basic checklist:
- Check project is built
- Check `psc-ide-server --version` matches compiler version and this is on the path

**Comment by nwolverson** [comment]
Did you ever get this to work, or are you still using the extension?

**Comment by texastoland** [comment]
This comes and goes for me. I'll update when I can reliably reproduce.

**Comment by spicydonuts** [comment]
Also having issues with this.  I open the project and everything works great, then as I hover over stuff it eventually stops working (in this case the first one to fail was `Option` from `purescript-options` after 3 others worked).  Since it's intermittent, it'd be nice if there was a quick command to restart `psc-ide-server`.

**Comment by spicydonuts** [comment]
I actually just found one thing that was breaking it, which is probably more on `psc-ide-server` than anything else, but still.  The Pux starter app has `npm run build` set to clean and then run webpack.  Clean removes `output` entirely and from that point on `psc-ide-server` is hosed, even though a few seconds later it rebuilds it.  Suggestions still stop working randomly, but much less frequently now that I've set the clean and webpack build commands to use `output_webpack` for their stuff.

One extra thing to note -- sometimes when it stops working it just shows "Loading..." when hovering over anything while other times it stops attempting to show tooltips at all.  Even ⌘K ⌘I won't show any sort of tooltip.
